linux/git_mirror

```diff
@@ -544,7 +544,12 @@ function git_mirror_repos()
   else
     GIT_MIRROR_MAIN_REPO="${1}"
   fi
-  local BRANCH="${2-master}"
+
+  local BRANCH="${2-}"
+  # Use git's init.defaultBranch default value. If it is unset, the default is still master in the latest git
```

Comment:
```suggestion
  # Use git's init.defaultBranch default value. If it is unset, the default is
  # still master in the latest git
```

Reply:
it looks like we wrap these kinds of comments (i.e., comments, as opposed to documentation)